examples/imagenet/dali_util.py

```diff
@@ -26,20 +26,20 @@ def _pair(x):
 class DaliPipelineTrain(pipeline.Pipeline):
 
     def __init__(self, file_list, file_root, crop_size,
-                 batch_size, num_threads, device_id,
+                 batch_size, n_threads, device_id,
```

Comment:
I didn't put a procedure to keep the backward compatibility. (the same as `VGG`)